scripts/package-harvester-os

```diff
@@ -14,7 +14,7 @@ source ${SCRIPTS_DIR}/version-rancher
 source ${SCRIPTS_DIR}/version-harvester ${TOP_DIR}/../harvester
 source ${SCRIPTS_DIR}/version-monitoring
 
-BASE_OS_IMAGE="rancher/harvester-os:20220406"
+BASE_OS_IMAGE="bk201z/harvester-os:20220406-60b74cd"
```

Comment:
Build succeeded and we can now replace it.

https://drone-publish.rancher.io/harvester/os2/46/1/2


```suggestion
BASE_OS_IMAGE="bk201z/harvester-os:20220407-ae38642"
```

Reply:
Need a tag on harvester/os2 repo

Reply:
Sure, The image is published and used.